docs/releases/1_7_0.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 1.7.0
+sidebar_position: 9950
+---
+
+# 1.7.0 - 2023-12-21
+
+_COMPATIBILITY NOTICE_  
+Starting in 1.7.0, the Airflow integration will no longer support Airflow versions `>=2.8.0`.  
+Please use the [OpenLineage Airflow Provider](https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/index.html) instead.
+
+### Added
+* **Airflow: add parent run facet to `COMPLETE` and `FAIL` events in Airflow integration** [`#2320`](https://github.com/OpenLineage/OpenLineage/pull/2320) [@kacpermuda](https://github.com/kacpermuda)  
+    *Adds a parent run facet to all events in the Airflow integration.*
+
+### Fixed
+* **Airflow: repair up.sh for MacOS** [`#2316`](https://github.com/OpenLineage/OpenLineage/pull/2316) [`#2318`](https://github.com/OpenLineage/OpenLineage/pull/2318) [@kacpermuda](https://github.com/kacpermuda)  
+    *Some scripts were not working well on MacOS. This adjusts them.*
+* **Airflow: repair `run_id` for `FAIL` event in Airflow 2.6+** [`#2305`](https://github.com/OpenLineage/OpenLineage/pull/2305) [@kacpermuda](https://github.com/kacpermuda)  
+    *The `Run_id` in a `FAIL` event was different than in the `START` event for Airflow 2.6+.*
+* **Flink: open Iceberg `TableLoader` before loading a table** [`#2314`](https://github.com/OpenLineage/OpenLineage/pull/2314) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
+    *Fixes a potential `NullPointerException` in 1.17 when dealing with Iceberg sinks.*
+* **Flink: name Kafka datasets according to the naming convention** [`#2321`](https://github.com/OpenLineage/OpenLineage/pull/2321) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
+    *Adds a `kafka://` prefix to Kafka topic datasets' namespaces.*
+* **Flink: fix properties within `JobTypeJobFacet`** [`#2325`](https://github.com/OpenLineage/OpenLineage/pull/2325) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
+    *Fixes properties assignment in the Flink visitor.*
+* **Spark: fix `commons-logging` relocate in target jar** [`#2319`](https://github.com/OpenLineage/OpenLineage/pull/2319) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
+    *Avoids relocating a dependency that was getting excluded from the jar.*
+* **Spec: fix inconsistency with Redshift authority format** [`#2315`](https://github.com/OpenLineage/OpenLineage/pull/2315) [@davidjgoss](https://github.com/davidjgoss)  
+    *Amends the `Authority` format for consistency with other references in the same section.*
+
+### Removed
+* **Airflow: remove Airflow 2.8+ support** [`#2330`](https://github.com/OpenLineage/OpenLineage/pull/2330) [@kacpermuda](https://github.com/kacpermuda)  
+    *To encourage use of the Provider, this removes the listener from the plugin if the Airflow version is `>=2.8.0`.*
```

Comment:
I would reformulate this in a way that points to the provider doc to not make people think it's overall removed

Reply:
Thanks @mobuchowski . I revised that description to add a link to the provider docs and, because we already have the compatibility notice at the top, say that the listener isn't imported instead. WDYT?